resources/views/hardware/view.blade.php

```diff
@@ -977,6 +978,7 @@
                                                     </td>
                                                     <td>{{ $component->pivot->assigned_qty }}</td>
                                                     <td>{{ Helper::formatCurrencyOutput($component->purchase_cost) }} each</td>
+                                                    <td>{{$component->serial}}</td>
```

Comment:
Can you fix the cuddling on those braces? `{{ $component->serial }}` vs `{{$component->serial}}`

Reply:
cuddlers have been spaced